server/handlers/client.js

```diff
@@ -0,0 +1,86 @@
+var dao = require("../dao/dao");
+const { Client } = require("../models/client");
+const {
+   clientIDPathValidator,
+   fullNameBodyValidator,
+   phoneNumberBodyValidator,
+    emailBodyValidator,
+    addressBodyValidator,
+    walletBodyValidator,
+  } = require("./shared_validators");
+  const { validationResult } = require("express-validator");
+  
+  exports.getClientByIDValidatorChain = [clientIDPathValidator];
+  
+  exports.getClientByIDHandler = async function (req, res, next) {
```

Comment:
you are using a wrong example of the handlers. As we said we changed because it was faulty in case of errors. If you see in the main branch my getClientByIDHandler you will understand easily how to implement handlers.